### ISSUE TITLE
New feature request (auto download feature)

### ISSUE THREAD

**Issue by dmswd** [gmathi/NovelLibrary#192]
When the library is being updated new chapters are automatically added to the download queue and start downloading when updating is finished

**Comment by gmathi** [comment]
Yes this is being worked on. Currently working towards enabling if the novels in a section should be auto synced or not. Then will be providing a button which says, auto download chapters for this novel. But for sure working on this.
